frontend/src/screens/apps/NewApp.tsx

```diff
@@ -349,7 +355,7 @@ const NewApp = () => {
                 Connection expiry time
               </p>
               <p className="text-gray-600 dark:text-gray-300 text-sm">
-                {new Date(parseInt(expiresAtParam) * 1000).toISOString()}
+                {new Date(expiresAtParam).toLocaleString()}
```

Comment:
here we should only render this if `expiresAtParam` is set, otherwise maybe show "never"?

Reply:
Done

Reply:
hmm, this was a bit wrong because that code wasn't even executed if expiresAt was set. Fixed now.